docker/pypi/wmagent-couchdb/Dockerfile

```diff
@@ -92,9 +68,17 @@ alias manage=$COUCH_MANAGE_DIR/manage
 export PS1="(CouchDB-$TAG) [\u@\h:\W]\$ "
 EOF
 
-RUN chown -R ${USER}:${GROUP} ${COUCH_ROOT_DIR}
+# set CouchDB docker specific bash prompt and manage alias for all users:
+RUN <<EOF cat >>/root/.bashrc
+alias manage=$COUCH_MANAGE_DIR/manage
+export PS1="(CouchDB-$TAG) [\u@\h:\W]\$([[ \$(id -u) -eq 0 ]] && echo \# || echo \$) "
+EOF
+
+RUN <<EOF cat >> ~/.bashrc
```

Comment:
I'll be pretty surprised if I see this to be properly propagated to the correct runtime user. It will go into the home directory of the default user which comes with the couchdb upstream image. 

I am really concerned about those comand prompts, and as long as we have the init.sh scirpt for the WMAgent container, I am calm for the wmagent (we can set those there and forget), but for mariadb and couchdb the closest thing we have is the runt.sh script, but this means reediting `~/.bashrc` constantly on every restart - not the best practice, to my opinion.

Reply:
Ok now,

So, @vkuznet  @amaltaro, I tested few things and here is the final solution to this annoying problem with the command  prompts for user agnostic containers. It was quite trivial at the end: 
https://github.com/dmwm/CMSKubernetes/pull/1467/commits/c7fec388edcc7c26b0a739411065a1465e6172d6

And if a dare to say.... we should expand this and move to `run.sh` all tweaks related to the `~./bashrc` file of the runtime estimated  user. I am talking about all those in particular: 
https://github.com/dmwm/CMSKubernetes/blob/f4a43f8a85bb1be3b59b913b59914f5f66014a34/docker/pypi/wmagent/install.sh#L153-L185 

Or as Alan did here with the `set_cronjob` function, to enclose these all tweaks in a separate function and call it at the end of `init.sh`, this would achieve the same result and at the same time keep the `run.sh` script as clean as possible.

Reply:
BTW, I started to reply in the CouchDB  container related changes when it came to those command prompts... but it is equally valid for the 3 types of containers. Please consider those arguments equally applied to the 3 of them. And the parts that relate the the `wmagent/install.sh` file of course are just an example of the same problem - i.e. not knowing at deploy time which would be the default user at runtime.

Reply:
Noted Todor. I am making these changes and will be back with updates soon. Thank you.